java-extension/com.microsoft.java.test.plugin/src/main/java/com/microsoft/java/test/plugin/util/TestSearchUtils.java

```diff
@@ -189,63 +193,66 @@ public static ASTNode parseToAst(final ICompilationUnit unit, IProgressMonitor m
         final Gson gson = new Gson();
         final SearchTestItemParams params = gson.fromJson((String) arguments.get(0), SearchTestItemParams.class);
 
-        final IJavaSearchScope scope = createSearchScope(params);
+        if (params.getLevel() == TestLevel.METHOD) {
+            // unreachable code since the client will directly run the test when it's triggered from method level
+            return Collections.emptyList();
```

Comment:
if it's unreachable, maybe throw excepetion or do assertion? (when unexpected scenario does occur, it would be fast located..)